script.js

```diff
@@ -1,4 +1,137 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+// Use Math to generate system's generated input for the game
+const getRandomHand = () => {
+  const systemGenerateNumber = 1 + Math.floor(Math.random() * 3);
+  if (systemGenerateNumber === 1) {
+    return "scissors";
+  } else if (systemGenerateNumber === 2) {
+    return "paper";
+  } else return "rock";
+};
+
+// Convert user's text input into symbol for output display
+const iconForNoun = (hand) => {
+  if (hand.toLowerCase() === "paper") {
+    return "🧻";
+  } else if (hand.toLowerCase() === "scissors") {
+    return "✂️";
+  } else if (hand.toLowerCase() === "rock") {
+    return "💎";
+  } else return hand.toLowerCase();
+};
+
+// all output messages
+const invalidMessage = (a) =>
+  `${iconForNoun(
+    a
+  )} is an invalid input.\<br>\Please type scissors/paper/rock to continue the game.`;
+const drawMessage = (a, b, c, d, e) =>
+  `Draw.\<br>\Your ${iconForNoun(a)} vs system's ${iconForNoun(
+    b
+  )}.\<br>\You: ${c}\<br>\System: ${d}.\<br>\Winning so far is ${e}.`;
+const winMessage = (a, b, c, d, e) => `You won.\<br>\Your ${iconForNoun(
+  a
+)} beats system's ${iconForNoun(b)}.\<br>\
+        You: ${c}\<br>\System: ${d}.\<br>\Winning so far is ${e}.`;
+const loseMessage = (a, b, c, d, e) =>
+  `You lost.\<br>\ System's ${iconForNoun(b)} beats your ${iconForNoun(
+    a
+  )}.\<br>\You: ${c}\<br>\System: ${d}.\<br>\Winning so far is ${e}.`;
+const koreanDrawMessage = (a, b, c, d, e) =>
+  `${drawMessage(a, b, c, d, e)}\<br>\ Ultimate winner is ${e}.`;
+const koreanWinMessage = (a, b, c, d, e) =>
+  `${winMessage(a, b, c, d, e)}\<br>\ You shout "muk-jji-ppa!".`;
+const koreanLoseMessage = (a, b, c, d, e) =>
+  `${loseMessage(a, b, c, d, e)}\<br>\ System shouts "muk-jji-ppa!".`;
+
+// All invalid cases of user's input
+const invalidCases = (a) =>
+  a.toLowerCase() !== "paper" &&
+  a.toLowerCase() !== "rock" &&
+  a.toLowerCase() !== "scissors";
+// All cases where user wins
+const winCases = (b, c) =>
+  (b.toLowerCase() === "paper" && c === "rock") ||
+  (b.toLowerCase() === "rock" && c === "scissors") ||
+  (b.toLowerCase() === "scissors" && c === "paper");
+// All cases where both parties choose the same object.
+const drawCases = (d, e) => d.toLowerCase() === e;
```

Comment:
Arrow functions are not covered in Basics. Though, again great desire to learn more and apply!
Like how you are keeping your code minimal, though to not overemphasise, do also consider readability i.e use more descriptive variables other than `d` or `e`